tests/IntegrationTestDefs.js

```diff
@@ -208,12 +208,12 @@ gpii.tests.dev.testTrayKeyedIn = function (tray, activePrefSet) {
 };
 
 gpii.tests.dev.testMultiPrefSetMenu = function (tray, menuTemplate) {
-    gpii.tests.app.testTemplateExists(menuTemplate, 15);
+    gpii.tests.app.testTemplateExists(menuTemplate, 9);
     gpii.tests.app.testItem(menuTemplate[0], "Open Morphic");
     // the default pref set should be set
-    gpii.tests.dev.testMultiContextKeyedIn(tray, menuTemplate, /*activeSetIdx=*/prefSetsInDevStartIdx);
-    gpii.tests.app.testItem(menuTemplate[13], "Reset Morphic");
-    gpii.tests.app.testItem(menuTemplate[14], "Exit GPII");
+    // gpii.tests.dev.testMultiContextKeyedIn(tray, menuTemplate, /*activeSetIdx=*/prefSetsInDevStartIdx); // Commented after GPII-3826
```

Comment:
I'm more a friend of deleting lines rather than commenting them in case it's still unclear to us by when they're going to be uncommented. Up to you, my vote goes for removing them if they can easily be re-added in the future and there aren't strong reasons for keeping them there.